site/styles.css

```diff
@@ -85,6 +86,18 @@ div.callout.callout {
   font-size: .8em;
 }
 
+.nav-item {
```

Comment:
With the padding being introduced here and in other places, the left sidebar navigation is starting to feel a bit squished. Would it maybe make sense to tinker with that a bit as well? It might not need much.

FWIW, I did try a few different screen resolutions and the padding introduced here works fine.